### PR TITLE
Upgrade UI to Vue3

### PR DESCRIPTION
**Pull request by rmotitsuki**
Related to https://github.com/kytos-ng/ui/issues/35

Summary
This PR updates the UI to the Vue3 compatibility framework.
It requires the latest UI to work.

### CODE REVIEW

**Review by Alopalao — CHANGES_REQUESTED** [review]
The buttons `Back to list` and `Redeploy EVC` are misplaced.
Screenshot_20240519_174626
Traditionally they are at the bottom.

**Review by italovalcy — CHANGES_REQUESTED** [review]
LGTM. Just left a few comments regarding the maximization of the infoPanels

File: ui/k-info-panel/list_connections.kytos [hunk 1]
```diff
@@ -391,4 +391,8 @@ module.exports = {
   display: inline-block;
   width: 12px;
 }
+
+.k-info-panel-max {
+  width: calc(100% - 305px);
+}
```
[the remark this answers is not in the document]
Comment (rmotitsuki) [reply ui/k-info-panel/list_connections.kytos]: Changed as requested. I added another calc to fill the screen.

File: ui/k-info-panel/show_circuit.kytos [hunk 2]
```diff
@@ -365,7 +357,7 @@ module.exports = {
           title: "View Connections",
```
[the remark this answers is not in the document]
Comment (rmotitsuki) [reply ui/k-info-panel/show_circuit.kytos:358]: Changed as requested.

**Review by Alopalao — APPROVED** [review]
LGTM

**Pull request merged**